packages/core/src/BackgroundImage/BackgroundImage.stories.tsx

```diff
@@ -80,3 +80,10 @@ Rounded.args = {
   rounded: 'top',
   height: '100px',
 }
+
+export const BackgroundPosition = Template.bind({})
+BackgroundPosition.args = {
+  height: 400,
+  width: '360px',
+  children: <StoryChildren />,
```

Comment:
should this have a background-position set in args?

Reply:
we can omit backgroundPosition from the args since it defaults to center

Reply:
The chromatic snapshot should probably picture something different from the default behaviour and I think that is based on the args passed in initially. Could we add `backgroundPosition: top` or something please

Reply:
yea sure